sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1 mb-12"
+  >
+    <template #actions>
+      <div class="search-box-display-width">
+        <v-text-field
+          label="Search name or application #"
+          prepend-inner-icon="mdi-magnify"
```

Comment:
mm, we use font awesome icons. are there any technical difficulties using font awesome?